app/views/projects/index.html.slim

```diff
@@ -28,6 +29,10 @@ table.table.table-striped.table-bordered.table-condensed.hidden-sm.hidden-xs
               = icon('minus')
       td = project_tags project
       td
+        - if project.teams.any?
+          - project.teams.each do |team|
+            = link_to team.name, activities_url(team_id: team.id)
```

Comment:
We only list [projects for the current season](https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/master/app/controllers/projects_controller.rb#L26)
So, adding the season name would seem redundant to me. Okay?
(Extra: I'd like to add the season name to the title though; that is part of [another issue](https://github.com/rails-girls-summer-of-code/rgsoc-teams/issues/784#issuecomment-316998327).)